app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -146,7 +146,10 @@ class HomeFragment : Fragment() {
     private val store: BrowserStore
         get() = requireComponents.core.store
 
-    private val onboarding by lazy { FenixOnboarding(requireContext()) }
+    private val onboarding by lazy { StrictMode.allowThreadDiskReads().resetPoliciesAfter {
+        FenixOnboarding(requireContext()) }
+    }
```

Comment:
nit: } }

We opened two braces on one-line so we should close to braces on one-line